.github/workflows/osx.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         # For testing 3.11 we need to do something like the below until the next Mathics3 is released
         # python -m pip install -e git+https://github.com/Mathics3/mathics-scanner#egg=Mathics-Scanner[full]
         # python -m pip install -e git+https://github.com/Mathics3/mathics-core#egg=Mathics3[full]
-        git clone --branch docpipeline_django_compat https://github.com/Mathics3/mathics-core
+        git clone --branch main https://github.com/Mathics3/mathics-core
```

Comment:
Of course, should be "master", not "main".

Reply:
Indeed...